App/Sagas/TextileSagas.ts

```diff
@@ -138,6 +138,53 @@ export function * handleProfilePhotoSelected(action: ActionType<typeof UIActions
   }
 }
 
+export function * handleProfilePhotoUpdated(action: ActionType<typeof UIActions.updateProfilePicture>) {
```

Comment:
Looks like lots of shared code with the `handleProfilePhotoSelected` method. Should we combine?